# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1229345351 -3600
#      Mon Dec 15 13:49:11 2008 +0100
# Node ID 096c22ce2b0b2bcc4c2a74102305f4c0404bc579
# Parent  0b7566aea6278dfb8a605f17e5b13d558dc65e39
style fixes in previous patch

diff --git a/scripts/optimization/lsqnonneg.m b/scripts/optimization/lsqnonneg.m
--- a/scripts/optimization/lsqnonneg.m
+++ b/scripts/optimization/lsqnonneg.m
@@ -116,18 +116,18 @@ function [x, resnorm, residual, exitflag
         x = xtmp;
       else
         ## LH8, LH9: find the scaling factor and adjust X.
         mask = (xtmp < 0);
         alpha = min (x(mask)./(x(mask) - xtmp(mask)));
         ## LH10: adjust X.
         x = x + alpha*(xtmp - x);
         ## LH11: move from P to Z all X == 0.
-	z |= (x == 0);
-	p = ~z;
+        z |= (x == 0);
+        p = ~z;
       endif
     endwhile
     w = c'*(d - c*x);
   endwhile
   ## LH12: complete.
 
   ## Generate the additional output arguments.
   if (nargout > 1)
